src/containers/shared/components/Amount.tsx

```diff
@@ -8,9 +8,14 @@ import { ExplorerAmount } from '../types'
 export interface AmountProps {
   value: ExplorerAmount
   displayIssuer?: boolean
+  modifier?: string
```

Comment:
nit: should we add a comment about the usage of this prop? may not be obvious at first glance

Reply:
Will do.